lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "lerna": "2.0.0-beta.32",
-  "version": "4.0.0-beta.8",
+  "version": "independent",
```

Comment:
Done as a separate commit (d296fa9d7245d1e0dc77dd55882f07ea17e9bbfb) so we can revert easily.